kremlib/kremlib.h

```diff
@@ -228,6 +228,12 @@ typedef const char *Prims_string;
 #define htobe32(x) BE_32(x)
 #define be32toh(x) BE_IN32(x)
 
+/* ... for the BSDs */
+#elif defined(__FreeBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
+#include <sys/endian.h>
+#elif defined(__OpenBSD__)
+#include <endian.h>
```

Comment:
@landryb, do you have `<endian.h>` on OpenBSD? [`/usr/src/include`](https://github.com/openbsd/src/tree/master/include/) suggests otherwise.

Reply:
`usr/include/endian.h` is a symlink to `usr/include/sys/endian.h` (cf https://github.com/openbsd/src/blob/master/sys/sys/endian.h) which says the following:
```
 * Public definitions for little- and big-endian systems.
 * This file should be included as <endian.h> in userspace and as
 * <sys/endian.h> in the kernel.
```